tests/src/rules/order.js

```diff
@@ -3648,6 +3648,24 @@ flowRuleTester.run('order', rule, {
         import typeof {foo} from 'common';
         import {bar} from 'common';
       `,
+    }),
+    test({
+      options: [
+        {
+          named: true,
+        },
+      ],
+      code: `
+        const test = {
+          a: 1,
+          browser: 2,
+        };
+
+        module.exports = {
+          ...test,
+          platform: 'node',
+        };
+      `,
```

Comment:
i'd like to also include a test that demonstrates that we only recognize `platform` as a named export from this file.

Reply:
Im not that familiar with this codebase. How could I test this?

Reply:
you'd want to make a fixture file with this test code in it, and then make two test cases that import this test file - one that imports `platform` and works, and one that imports `a` and/or `browser` and fails.

Reply:
Im very sorry but can't figure it out. I added the cases but it doesn't fail? I suspect that I'm doing something wrong but cant figure out what.

Reply:
Can you push up what you have? I'll take a look.

Reply:
I tried looking at #3032 as it had fixture files I believe

Reply:
@ljharb did you have a chance to look?

Reply:
?